src/index.html

```diff
@@ -4,4 +4,17 @@
 <Body:>
   <div class="p2">
     <view is="{{$render.ns}}"/>
+    
+    {{if $render.url !== "/"}}
+      <div class="p2 row">
+        <div class="col-12">
+          <hr/>
+          <p><a href="/">Back to Welcome Page</a></p>
```

Comment:
I think some of the components like the `racer-basics` are using the `col-6` class in a way that sometimes causes this link to not appear in the correct area 🤔  - happy to pair on this if you wanted to dig into this further